programs/pluto/ikev2_ts.c

```diff
@@ -1655,6 +1656,12 @@ bool process_v2TS_request_payloads(struct ike_sa *ike,
 			     best.connection->name, pri_so(best.connection->serialno),
 			     (is_from_group(best.connection) ? " from group" : ""),
 			     cc->name, pri_so(cc->serialno));
+
+			if (has_outstanding_ike_auth_request(best.connection, ike, md)) {
+				llog_ts(child, &tsps, "best connection matching TS has IKE_AUTH request outstanding");
+				return false;
```

Comment:
@cagney Hm, this actually leads to childless IKE SA but is there a way to drop IKE SA at this point?